docs/release_notes/pending_release.rst

```diff
@@ -1,10 +1,16 @@
-SMQTK Pending Release Notes
-===========================
+Pending Release Notes
+=====================
 
+This is the initial release of ``smqtk-indexing``, spinning off from
+v0.14.0 of the monolithic `SMQTK`_ library.
+
+.. _smqtk: https://github.com/kitware/smqtk
 
 Updates / New Features
 ----------------------
 
+Misc.
+* Fixing documentation and adding RTD
```

Comment:
I believe reST will complain about there not being whitespace between a bullet and the previous line.
```suggestion
Misc.

* Fixing documentation and adding RTD
```